tests/functional/common.js

```diff
@@ -90,6 +90,12 @@ export type RunningCommand = {|
 export function execCommand(
   execPath: string, argv: Array<string>, spawnOptions: child_process$spawnOpts,
 ): RunningCommand {
+  // bin/web-ext can't be directly spawned on Windows
+  if (execPath === webExt && process.platform === 'win32') {
+    argv.unshift(execPath);
+    execPath = process.execPath;
+  }
+
   const spawnedProcess = spawn(execPath, argv, spawnOptions);
```

Comment:
if you change this to:

    const spawnedProcess = spawn(process.execPath, webExt, argv, spawnOptions);

won't it work for all platforms?

Every call to it begins with `execCommand(webExt...)` so we could maybe just rename it to `execWebExt()`.